randomerz/Slider
# Adds Magitech Portal SFX (in Main scene's AudioManager this time)

## Description

Adds Magitech Portal SFX

## Related Task

Polish: SFX

## How Has This Been Tested?



- Run Magitech Scene and enter the portal
- Observe the portal sound

## Screenshots (if appropriate):
